functions-and-operators/string-functions.md

```diff
@@ -115,11 +115,35 @@ Return the character for each integer passed.
 
 ### [`CHAR_LENGTH()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_char-length)
 
-Return number of characters in argument.
+The `CHAR_LENGTH()` function is used to get the number of characters in a specified expression. It returns the total number of strings in a specified expression as an integer value. 
+
+Examples:
+
+```sql
+SELECT CHAR_LENGTH("TiDB") AS LengthOfString;
+
++----------+
+| LengthOfString |
++----------+
+| 4  |
++----------+
+```
+
+```sql
+SELECT CustomerName, CHAR_LENGTH(CustomerName) AS LenghtOfName FROM Customers;
+
+| CustomerName|LengthOfName  |
+|-------------|--------------|
+| Albert Einstein | 15 |
+| Robert Oppenheimer | 18 |
```

Comment:
Maybe include a name where `CHAR_LENGTH()` and `LENGTH()` don't match:
```
mysql> WITH n AS (
    -> SELECT 'François Englert' AS p
    -> UNION ALL SELECT 'Albert Einstein'
    -> UNION ALL SELECT 'Robert Oppenheimer'
    -> )
    -> SELECT p, CHAR_LENGTH(p), LENGTH(p) FROM n;
+--------------------+----------------+-----------+
| p                  | CHAR_LENGTH(p) | LENGTH(p) |
+--------------------+----------------+-----------+
| François Englert   |             16 |        17 |
| Albert Einstein    |             15 |        15 |
| Robert Oppenheimer |             18 |        18 |
+--------------------+----------------+-----------+
3 rows in set (0.00 sec)
```

Reply:
I think this should be introduced in the `LENGTH()` subheading where the difference between `LENGTH()` and `CHAR_LENGTH()` will be stated and this type of example will be included. Adding it here when `LENGTH()` is yet to be introduced to the readers can confuse.